components/editor/prosemirror-plugins.md

```diff
@@ -0,0 +1,134 @@
+---
+title: ProseMirror Plugins
+page_title:  ProseMirror Plugins
+description: Explore how to use the ProseMirror Plugins in the Editor for Blazor uses.
+slug: editor-prosemirror-plugins
+tags: telerik,blazor,editor,prosemirror,plugins
+published: True
+position: 130
+---
+
+# ProseMirror Plugins
+
+The Telerik UI for Balzor Editor component is based on the [ProseMirror library](https://prosemirror.net/). ProseMirror provides a set of tools and concepts for building rich text editors, using user interface inspired by what-you-see-is-what-you-get.
+
+## Concept
```

Comment:
Consider switching to `Plugins Concept` to match what you did in the schema overview with `Schema Concept`